tests/unit/list/__snapshots__/list.spec.js.snap

```diff
@@ -25,7 +25,7 @@ exports[`List #render() should not modify mods 1`] = `
 </ul>
 `;
 
-exports[`List #render() should not render empty items if an appropriate prop is false 1`] = `
+exports[`List #render() should not render empty items 1`] = `
```

Comment:
Is this `1` at the end of the message a typo?

Reply:
No... that's the snapshot creator numbering the snapshot check